lib/schema.js

```diff
@@ -1206,6 +1206,11 @@ Schema.prototype.queue = function(name, args) {
  *       console.log(this.getFilter());
  *     });
  *
+ *     // Equivalent to calling `pre()` on `save`, `findOneAndUpdate`.
+ *     toySchema.pre(['save', 'findOneAndUpdate'], function(next) {
+ *       console.log(this.getFilter());
```

Comment:
`this.getFilter()` won't work on `pre('save')`. Perhaps make this `toySchema.pre(['updateOne', 'findOneAndUpdate'])`?

Reply:
Thanks for correcting me, I created a new commit.